chrome/android/java/AndroidManifest.xml

```diff
@@ -16,7 +16,7 @@ by a child template that "extends" this file.
     tools:ignore="MissingVersion">
     <!-- android:versionCode and android:versionName is set through gyp. See build/common.gypi -->
 
-    <uses-sdk android:minSdkVersion="{{min_sdk_version}}" android:targetSdkVersion="{{target_sdk_version}}" tools:overrideLibrary="org.adblockplus.libadblockplus.android.settings"/>
```

Comment:
Do I correctly understand that *settings* required some more advanced version of the sdk than the one which is used here, and since it's not the case anymore, we can remove the attribute `tools:overrideLibrary`?

Reply:
No. Builder controls app's `minSdkVersion` is higher or equal than highest library's `minSdkVersion`. Since we had >21 limitation in `libadblockplus-settings` module we had to override it and still set 16 as min. Now we just lowered it in `libadblockplus-settings` and we don't have to override it anymore.